nasa-gibs/worldview
# WV-2281: Standalone about page

## Description

- Add a URL param to indicate if about page is open or not
- Clean up some places where "Worldview" was hard-coded into the about sections

## How To Test

* Load http://localhost:3000/?abt=on
* Confirm about page shows
* Close about page
* Refresh page
* Confirm about page does not show
